engine/src/main/java/org/terasology/world/block/entity/neighbourUpdate/NeighbourBlockFamilyUpdateSystem.java

```diff
@@ -69,6 +71,17 @@ public void largeBlockUpdateFinished(LargeBlockUpdateFinished event, EntityRef e
         }
     }
 
+    @ReceiveEvent(components = {BlockItemComponent.class})
+    public void onPlaceBlock(OnBlockItemPlaced event, EntityRef entity) {
```

Comment:
so this was for the placement problems where a block is placed but the neighboring blocks never update. I think there might be a problem with BlockItemComponent.class because I believe this would never call if the block does not have an item tied to it.

So would I do this instead:
@ReceiveEvent()
 public void onPlaceBlock(OnBlockItemPlaced event, EntityRef entity) {
...